### ISSUE TITLE
"Replace with" action removes ticks from TemplateHaskellQuotes usage

### ISSUE THREAD

**Issue by expipiplus1** [haskell/haskell-language-server#628]
Oops, I've misspelled `break`

```haskell
{-# LANGUAGE TemplateHaskellQuotes #-}

module Foo where

import Language.Haskell.TH (Name)

foo :: Name
foo = 'bread
```

HLS suggests `Replace with ‘break’` (weird quotes btw), perfect:

```haskell
foo :: Name
foo = break
```

oh no, `Couldn't match expected type ‘Name’ with actual....`

**Comment by jneira** [comment]
mmm I guess this one will not be too difficult, it could be related with the fact that in windows for older ghcs the identifiers in error msgs was surrounded by 'id```

**Comment by sloorush** [comment]
Hey @jneira, I was trying to find a solution to this issue; I have managed to successfully use my hacked HLS. 

Could you guide me on where to look to solve this issue?

**Comment by jneira** [comment]
@sloorush hi, thanks for working in a fix for this, from 

>  I have managed to successfully use my hacked HLS.

i understand you have a branch with the fix, in that case i would open a pr against this repo master branch, noting it will close this issue

**Comment by sloorush** [comment]
By that I mean i was successfully able to use my custom build of HLS but I am unable to understand how to solve this issue.

It would be really helpful if you could guide me on how to solve it.

**Comment by bendo** [comment]
Will look at this issue.

**Comment by expipiplus1** [comment]
Thanks @bendo

**Comment by fendor** [comment]
fixed by #3260
